src/com/projectkorra/projectkorra/airbending/combo/AirSweep.java

```diff
@@ -198,7 +199,7 @@ public void manageAirVectors() {
 					if (!entity.equals(this.player) && !(entity instanceof Player && Commands.invincible.contains(((Player) entity).getName()))) {
 						if (this.knockback != 0) {
 							final Vector force = fstream.getLocation().getDirection();
-							GeneralMethods.setVelocity(entity, force.clone().multiply(this.knockback));
+							GeneralMethods.setVelocity(this,entity, force.clone().multiply(this.knockback));
```

Comment:
Please change to `GeneralMethods.setVelocity(this, entity, force.clone().multiply(this.knockback));`